static/js/stat_var_hierarchy/stat_var_hierarchy.tsx

```diff
@@ -248,9 +254,15 @@ export class StatVarHierarchy extends React.Component<
       if (this.props.selectSV) {
         this.props.selectSV(sv);
       }
-      this.setState({
-        svPath: Object.assign({ [sv]: path }, this.state.svPath),
-      });
+      if (this.props.type === StatVarHierarchyType.MAP) {
```

Comment:
nit: assign variables in if else, and have only one this.setState()

Reply:
done